inc/extras.php

```diff
@@ -35,8 +35,7 @@ function twentyseventeen_body_classes( $classes ) {
 	}
 
 	// Add class if no custom header or featured images
-	$header_image = get_header_image();
-	if ( '' == $header_image && ( ! has_post_thumbnail( $post->ID ) || is_home() ) ) {
+	if ( false === has_header_image() && ( ! has_post_thumbnail() || is_home() ) ) {
```

Comment:
Strict checks are only really useful/necessary when a function can return different types, e.g. a bool or a string, depending on context. Since `has_header_image()` only returns a bool, a simpler check should be preferred.